src/style.css

```
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+
+  --color-active: #00ACDC;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+  align-items: center;
+  height: 60px;
+}
+
+.nav__logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+}
+
+nav {
```

```suggestion
.nav {
```
Use classes selectors